cli-output/src/cli_output.rs

```diff
@@ -1149,6 +1150,51 @@ impl fmt::Display for CliFees {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CliTokenAccount {
+    pub address: String,
+    #[serde(flatten)]
+    pub token_account: UiTokenAccount,
+}
+
+impl fmt::Display for CliTokenAccount {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        writeln!(f)?;
+        writeln_name_value(f, "Address:", &self.address)?;
+        let account = &self.token_account;
+        println!("decimals: {}", account.token_amount.decimals);
+        println!("amount: {}", account.token_amount.amount);
```

Comment:
Shouldn't be `println`; but maybe remove these lines altogether?

Reply:
Oops!  Yep these were for debug 😅